admission-webhook/integration_tests/templates/credspec-with-hostagentconfig.yml

```diff
@@ -0,0 +1,26 @@
+# a sample cred spec
+
+apiVersion: windows.k8s.io/v1alpha1
+kind: GMSACredentialSpec
+metadata:
+  name: {{ index .CredSpecNames 0 }}
+credspec:
+  ActiveDirectoryConfig:
+    GroupManagedServiceAccounts:
+    - Name: WebApplication2
+      Scope: CONTOSO
+    - Name: WebApplication2
+      Scope: contoso.com
+    HostAccountConfig:
+      PluginGUID: "{GDMA0342-266A-4D1P-831J-20990E82944F}"
+      PluginInput: "contoso.com:gmsaccg:<password>"
```

Comment:
On Azure, the format of PluginInput is ‘ ObjectId=${msiObjectId};SecretUri=${secretUri}’

Reply:
I am not sure whether this plugin supports other input parameters for now 🙂.  It should be ok if it is only for test purpose.